zenoh-dds-interop/shapes_demo/README.md

```diff
@@ -0,0 +1,73 @@
+# shapes-demo
+
+A program to test the interoperability of [zenoh-plugin-dds](https://github.com/eclipse-zenoh/zenoh-plugin-dds) against various DDS implementations.
+
+## Dependencies
+
+
+- [Rust](http://rust-lang.org) installation instructions are available [here](https://www.rust-lang.org/tools/install)
+- [zenoh-plugin-dds](https://github.com/eclipse-zenoh/zenoh-plugin-dds)
+- A DDS Shapes Demo application, using e.g. [RustDDS](https://atostek.com/en/services/rust-dds/) that has a built-in [Shapes Demo](https://github.com/jhelovuo/RustDDS/tree/master/examples/shapes_demo).
```

Comment:
It would be better to remove the link pointing to the commercial page of `RustDDS`.

Reply:
Agreed. @jhelovuo could you please set https://github.com/jhelovuo/RustDDS as URL for `RustDDS`.
Also please add https://github.com/atolab/dds-ishapes as another DDS Shapes Demo using CycloneDDS.

Reply:
Thanks you!